src/plugins/discover/public/components/doc_table/_doc_table.scss

```diff
@@ -4,6 +4,7 @@
 // stylelint-disable selector-no-qualifying-type
 .kbnDocTableWrapper {
   @include euiScrollBar;
+  @include euiOverflowShadow;
```

Comment:
Added an overflow shadow to the legacy grid to improve the look of it with the new tabs design. It also looks good in the embeddable IMO.